examples/accounts/accounts.repl

```diff
@@ -46,18 +46,19 @@
 
 ; initiate pact with normal execution
 (payment "123" "us" "456" "them" 12.0 (time "2016-07-22T11:26:35Z"))
+(pact-state)
 (expect "balance of 123 after debit step" 217.0 (with-read accounts "123" { "balance" := b } b))
 
 ; test rollback
-(continue-pact 6 0 true)
+(continue-pact "Tug6q-wmaru1QqUMjm7RiwhnsQMTRcMHus65HwF7laE" 0 true)
```

Comment:
Since we're breaking `continue-pact`, let's finish the job and make it easier to use.
`(continue-pact STEP)` uses the pact id in the current `pact-state` value.
`(continue-pact STEP ROLLBACK)` same.
I don't think we need YIELD, but take a look. If necessary we can add setting YIELD in `pact-state` but I don't think it's used anywhere currently, in which case we drop it.
Thus the full version is `(continue-pact STEP ROLLBACK PACT-ID)` for those rare scenarios where you need to actually specify the id.

Reply:
None of the tests make use the yield section of `continue-pact`

Reply:
That should definitely be tested @LindaOrtega. It would be good to add one or two.

Reply:
I kept "yield" in `continue-pact` but made it a fourth overloading so its a little better to use, aka
```
(continue-pact STEP)
(continue-pact STEP ROLLBACK)
(continue-pact STEP ROLLBACK PACT-ID)
(continue-pact STEP ROLLBACK PACT-ID YIELD)
```
Also added a simple test in `yield.repl` that makes use of it. @emilypi let me know if I should add more tests.

@slpopejoy, I mainly kept it in `continue-pact` instead of adding it to `(pact-state)` because `(pact-state)` does not default to the last pact initiated across all transactions, but instead it defaults to the last pact initiated in the current transaction. So if its a new transaction and a pact was initiated in a previous transaction, running `pact-state` will return a `PactExec not in context` error. I assumed we would like to test overwriting yield across any pacts initiated across different transactions, not just the ones in the current transaction.